packages/audio-block/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "audio-block",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "serve": "frontify-cli block serve --entryPath src/index.ts",
+        "deploy": "frontify-cli block deploy --entryPath src/index.ts",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "typecheck": "tsc --noEmit"
+    },
+    "devDependencies": {
+        "@babel/core": "7.20.2",
+        "@frontify/eslint-config-react": "0.15.5",
+        "@frontify/frontify-cli": "5.3.1",
+        "@types/react": "17.0.50",
+        "@types/react-dom": "17.0.17",
+        "autoprefixer": "10.4.13",
+        "cypress": "11.0.1",
+        "eslint": "8.27.0",
+        "eslint-plugin-notice": "^0.9.10",
+        "postcss": "8.4.19",
+        "prettier": "2.7.1",
+        "tailwindcss": "3.2.3",
+        "typescript": "4.8.4"
+    },
+    "dependencies": {
+        "@frontify/app-bridge": "3.0.0-beta.38",
+        "@frontify/fondue": "12.0.0-beta.105",
+        "@frontify/fondue-tokens": "3.2.0",
+        "@frontify/guideline-blocks-settings": "0.25.10",
+        "@frontify/guideline-blocks-shared": "workspace:*",
+        "react": "17.0.2",
+        "react-dom": "17.0.2"
+    }
```

Comment:
updated deps [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/9687fbec09c7cebabb5d7539085a8022e4567459)

app-bridge and guideline-block-settings isn't possible to update otherwise the typecheck on other blocks will fail